### DISCUSSION TITLE
JSONFile radio button help

### DISCUSSION THREAD

**Discussion by ScottEKendall** [swiftDialog/swiftDialog#274] [Q&A] [answered]
I am trying to use a JSONFile to do dynamically construct & display options for the end users.  This is my JSONtest.json  input file:

```
{
  "radio" : [
      {"label" : "Update Now", "checked" : true, "disabled" : false },
      {"label" : "Defer 1 Day", "checked" : true, "disabled" : false },
      {"label" : "Defer 5 Days", "checked" : true, "disabled" : false },
      {"label" : "Defer 10 Days", "checked" : true, "disabled" : false }
      ]
}
```
and this is my terminal command: `dialog --title "Nudge Replacement" --jsonfile JSONtest.json`

But I am not getting the desired output (not getting any options actually)...what am I missing here?

**Comment by bartreardon** [comment, marked as answer]
not sure where the basis of that json came from. radio is a special type of select item and the json to generate what you want would look something like:

```json
{
  "selectitems" : [
      {
	"title" : "Update", 
	"values" : [
		"Update Now",
		"Defer 1 Day",
		"Defer 5 Days",
		"Defer 10 Days"
	],
	"style" : "radio"
      }
}
```
